scripts/commit_classifier.py

```diff
@@ -383,7 +383,7 @@ def load_user(phid):
                 message = replace_reviewers(message, reviewers)
 
             logger.info(
-                f"Applying {patch.phid} from revision {revision['id']}: {message}"
+                "Applying %s from revision %s: %s", patch.phid, message, revision["id"]
```

Comment:
The order here is not correct:
```suggestion
                "Applying %s from revision %s: %s", patch.phid, revision["id"], message
```